test/plugins/mongodb-core.spec.js

```diff
@@ -131,6 +131,47 @@ describe('Plugin', () => {
             }, () => {})
           })
 
+          it('should sanitize BSON as values and not as objects', done => {
```

Comment:
I.e. could that affect the workings of the tracer ?

Reply:
Yes, and I assumed the underlying buffer would be returned, but it seems this is not the case (which makes sense since a buffer is not valid JSON).

https://github.com/mongodb/js-bson/blob/10e5f005e69af242967c25c8d101fb61166f0ee2/lib/binary.js#L267

Should I make the change or add proper support for this use case in another PR?

Reply:
Separate PR is ok 👍